twitterscraper/tweet.py

```diff
@@ -38,7 +39,8 @@ def from_soup(cls, tweet):
                 'span', 'ProfileTweet-action--favorite u-hiddenVisually').find(
                     'span', 'ProfileTweet-actionCount')['data-tweet-stat-count'] or '0',
             html=str(tweet.find('p', 'tweet-text')) or "",
-        )
+            reply_to_id = tweet.findChildren()[0]['data-conversation-id'] or '0',
```

Comment:
This can also be achieved with 
`reply_to_id = tweet.find('div', 'tweet')['data-conversation-id'] or '0'`